src/components/templates/Search/utils.ts

```diff
@@ -68,6 +69,12 @@ export function getSearchQuery(
 
   const nestedQuery = {
     must: [
+      {
+        query_string: {
+          default_field: 'service.attributes.additionalInformation.isCompliant',
+          query: !!isCompliant
```

Comment:
Probably compliantOnly is a better name for the filter.

Also I don't like boolean for these types I guess. How do i distinguish between only non-compliant, complaint and both? Let's leave it for now, given the amount of time we have.

Reply:
I already had some doubts about the naming, because I wanted to match it with the application patterns  and I wasn't sure if isInCompliance would be better than isCompliant.

Reply:
I'm not sure if I understand what compliantOnly means...